examples/tutorials/darts.py

```diff
@@ -237,11 +238,13 @@ def evaluate_model(model, cuda=False):
 # one-shot strategy combines search with model training into a single run.
 # Compared to multi-trial strategies, one-shot NAS doesn't need to iteratively spawn new trials (i.e., models),
 # and thus saves the excessive cost of model training.
-# It's worth mentioning that one-shot NAS also suffers from multiple drawbacks despite its computational efficiency.
-# We recommend
-# `Weight-Sharing Neural Architecture Search: A Battle to Shrink the Optimization Gap <https://arxiv.org/abs/2008.01475>`__
-# and
-# `How Does Supernet Help in Neural Architecture Search? <https://arxiv.org/abs/2010.08219>`__ for interested readers.
+#
+# .. note::
```

Comment:
I believe a blank line after `.. note::` is required to make it work. But I'm not sure.

Reply:
I have tested it and found my way also works. But I will fix this to be consistent with your style.